Src/BootCamp.Chapter/Program.cs

```diff
@@ -6,7 +6,39 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            // Read name, surename, age, weight (in kg) and height (in cm) from console.
+            Console.Write("Please tell me your first name: ");
+            string name = Console.ReadLine();
+            Console.Write("Please tell me your surname: ");
+            string surname = Console.ReadLine();
+            Console.Write("Please tell me your age: ");
+            int age = int.Parse(Console.ReadLine());
+            Console.Write("Please tell me your weight in Kg: ");
+            int weight = int.Parse(Console.ReadLine());
+            Console.Write("Please tell me your height in cm: ");
+            double height = double.Parse(Console.ReadLine());
```

Comment:
weight and height should probably be of the same type.
float or double fit perfect here.

Reply:
fixed it thanks for the tip!